components/nexus-repository/src/main/java/org/sonatype/nexus/repository/proxy/ProxyFacetSupport.java

```diff
@@ -215,6 +215,7 @@ protected Content fetch(String url, Context context, Content stale) throws IOExc
       result.getAttributes().set(Content.CONTENT_LAST_MODIFIED,
           extractLastModified(request, response.getLastHeader(HttpHeaders.LAST_MODIFIED)));
       result.getAttributes().set(Content.CONTENT_ETAG, extractETag(response.getLastHeader(HttpHeaders.ETAG)));
+      indicateVerified(context);
```

Comment:
This is premature, isn't it? At this point, the Content hasn't been stored.

In any case, I think this logic should be handled by the format-specific store() method, to avoid a separate Orient transaction.